### ISSUE TITLE
Public domain POSIX make

### ISSUE THREAD

**Issue by Curculigo** [iglunix/iglunix#126]
https://github.com/rmyorston/pdpmake

> In the future, with a rewrite of GNU Make, we will have no GNU software at all.

Maybe you could send patches to pdpmake or even fork from it.

**Comment by anoraktrend** [comment]
pdpmake can't build the linux kernel.

**Comment by Curculigo** [comment]
> pdpmake can't build the linux kernel.

Yes. This is the reason why I said you need to send patches to the developer or even fork it. I think it's still better than writing your own implementation of gmake from scratch.

**Comment by Curculigo** [comment]
Btw, you could try to patch bmake or schilytools also has various implementations of make for you to choose.

**Comment by firasuke** [comment]
> pdpmake can't build the linux kernel.

I am seconding this and adding that `pdpmake` can't build a lot of things..
